lib/screens/ledger_view.dart

```diff
@@ -17,32 +17,32 @@ class LedgerViewArguments {
 
 class LedgerView extends HookWidget {
   const LedgerView({
-    Key? key,
+    super.key,
     this.ledger,
-  }) : super(key: key);
+  });
   final Ledger? ledger;
 
   @override
   Widget build(BuildContext context) {
-    var _ledger = useState<Ledger?>(null);
+    var viewLedger = useState<Ledger?>(null);
```

Comment:
Later, in this case where the same variable names occur, I think `ledgerState` is more readable.